source/_docs/assuming-write-access.md

```diff
@@ -43,7 +50,22 @@ The best solution is to communicate with the maintainer of the module or plugin
 7. Deploy to Live and perform the plugin operation that creates the desired files, then confirm results.
 
 ## Troubleshooting
+
+### Modules That Verify Directories
+
 Some modules and plugins verify that the target directory exists using `is_dir()` which returns bool(false) if the directory is a symlink. It may help to patch the module/plugin to use `is_link()` instead of `is_dir()`.
 
-## See Also		
-For more details on creating symbolic links on Mac/Linux, see [this thread](https://apple.stackexchange.com/questions/115646/how-can-i-create-a-symbolic-link-in-terminal).		
+### Incorrect Symlink Paths
+
+If a symlinked folder doesn't show the proper contents, doublecheck that the path is correct. In Bash, `ls -l` will show symlinks paths:
+
+```shell
+user@hostname:~/projects/myawesomesite/wp-content/uploads$ ls -l
```

Comment:
Until we formalize a style for displaying terminal output, can we stick to the existing standard that uses `$` to denote the prompt? Otherwise, looks good. +1